src/eepromfs.c

```diff
@@ -0,0 +1,933 @@
+/**
+ * @file eepromfs.c
+ * @brief EEPROM Filesystem
+ * @ingroup eepfs
+ */
+#include <stdio.h>
+#include <string.h>
+#include <stdint.h>
+#include <malloc.h>
+#include <sys/stat.h>
+#include "libdragon.h"
+#include "system.h"
+
+#define divide_ceil( x, y ) ((x / y) + (x % y != 0))
```

Comment:
`#define divide_ceil( x, y ) (((x)/(y)) + ((x) % (y) != 0))` for safer precedence